apps/explorer/src/components/recent-packages-card/RecentPackagesCard.tsx

```diff
@@ -52,11 +52,11 @@ const transformTable = (events: SuiEvents) => ({
             timestamp,
             txDigest,
         }: SuiEventEnvelope): PackageTableData => {
-            if (!('publish' in event)) return {};
+            if (!(event.type === 'publish')) return {};
```

Comment:
can we add an `isEventType(e: SuiEvent, type: 'publish'|'pay_sui'| 'transfer'....)` or `getPublishedEvent(e: SuiEvent): PublishedEvent | undefined` to the sdk and use it in the explorer, such that we don't have to make these changes everywhere if the schema change again?